src/vs/workbench/api/common/extHostTypeConverters.ts

```diff
@@ -2796,7 +2796,7 @@ export namespace ChatPromptReference {
 			range: variable.range && [variable.range.start, variable.range.endExclusive],
 			value: isUriComponents(value) ? URI.revive(value) :
 				value && typeof value === 'object' && 'uri' in value && 'range' in value && isUriComponents(value.uri) ?
-					Location.to(revive(value)) : value,
+					Location.to(revive(value)) : variable.isImage ? new types.ChatReferenceBinaryData('image/*', new Uint8Array(Object.values(value))) : value,
```

Comment:
Make sure to get the proper mime here instead of the generic `image/*` one

Reply:
yessir, waiting for https://github.com/microsoft/vscode/pull/229779 which let's us get the correct type (previously, it was almost always converted into `image/png`)